### ISSUE TITLE
Upload File Does Nothing

### ISSUE THREAD

**Issue by mdrobbins** [ExploreConsulting/NetSuite-Interface-Uploader#11]
Running VSCode on Windows with this extension installed.  I have:

* Created a new role with exactly the permissions outlined in the link referenced in the readme
* Added credentials (master password, email, password, role)
* Selected credentials
* Selected the root folder (or not since it defaults the SS folder).

When running the Upload File command either from command pallette or keyboard shortcut, the popup shows "Sending 'filename' to NetSuite..." and goes away after a few seconds.  The file is never uploaded.  There are no errors displayed in VSCode and there are no entries in the Web Service Usage logs in NetSuite.

I've tried multiple accounts, multiple roles, windows and mac, uninstalling/reinstalling the extension, and purging credentials and starting over.  I have the same issue in each case.

Any ideas?

**Comment by EmilChoparinov** [comment]
Hi @mdrobbins, that usually happens when the role set isn't set properly. Unfortunately the error messages received from NS are not very helpful to diagnose it. But the only way I've been able to replicate this is through an invalid role.

**Comment by EmilChoparinov** [comment]
Did a patch that will propagate errors to the extension, try diagnosing now

**Comment by mdrobbins** [comment]
Thanks for that.  I still don't see anything different.  Here's a [short video](https://youtu.be/zJeyMDP5qMY) that shows my role setup and walks through the plugin configuration to upload a file.

**Comment by EmilChoparinov** [comment]
Thanks for the video, it was really helpful. 

At first glance it looks like you dont have a folder in VSCode open. Try opening the folder that contains the file inside VSCode. My extension bases the open folder as root of whatever folder you set in NS.

I'll add some better error handling on my end. 

If you open the file in a folder and its still causing issues, I'll investigate that 'join' error that you've been receiving.

Thanks

**Comment by mdrobbins** [comment]
I did have a folder open.  I was just trying to simplify everything as much as I could so that was a new folder with just a single file in it.

Thanks again, for looking into this.

**Comment by EmilChoparinov** [comment]
I reviewed the video you sent closely and noticed no significant differences between what I did and you did. The only thing striking out at me is that "personal" dropdown, how did you get that to appear? Following what you did and the exact fields you had I was unable to replicate it.

Thanks for helping out with my questions and investigation

**Comment by mdrobbins** [comment]
The only thing I see that says personal in the video is the name of the folder that I had open at the time, a reference that it was my personal NetSuite instance.

**Comment by mdford** [comment]
@mdrobbins were you able to get this to work?  I am having similar issues.  I can obviously connect to NS because it gives me the list of my accounts to select from.  But, as you experienced, is "appears" to upload, the message goes away, no error messages, etc but nothing is ever uploaded to NS.  Also tried all of accounts, etc and I have full access rights on everything.  Hoping you (or @EmilChoparinov ) were able to find something causing the issues....

**Comment by EmilChoparinov** [comment]
I have yet to be able to replicate this issue. My best bet is that it might be due to some conflicting extensions. @mdford

**Comment by mdrobbins** [comment]
Yes, I still have this issue.  I've just tried removing all of my extensions except this one.  I've also uninstalled and reinstalled both the user- and system-installers for vscode and it doesn't seem to upload files.  Not sure what could be causing this with multiple NS account on both Windows and Mac for me.

**Comment by reyesmarq** [comment]
Is there any update on this issue?

**Comment by raytetlow** [comment]
I am getting the exact same issue. says its uploading. nothing happens.

**Comment by TexasSwede** [comment]
I have the same issue, ut says that the file is being uploaded, but it is not showing up on the server. No error message.
I have set the master password, I have connected to the correct account, where I have administrator access, and I have set the root folder to -15 (the internal ID for the SuiteScripts folder).
Is there anything I can do to help troubleshoot this issue? I am very excited about this VS Code extension, since it is very promising, so I would love to get it to work.

**Comment by dreamgear** [comment]
same behavior.. I'm using the same role I created for webstorm, so I know it's right.  The lack of any error message at all is not confidence-inspiring.

**Comment by amandamaddox3** [comment]
same behavior.  I am running Windows 10 Pro, Visual Studio Code 1.58.2

From the command pallet I have gone through the NSI: Add Account and filled out all of the prompts.   This step clearly works because it asks at the end to select information that came from the NetSuite server (not data that I entered).

Then when I run the NSI: Upload File and info popup displays "Sending <file> to NetSuite..."  
It takes a long time, then goes away.

When I log into the file cabinet on NetSuite and look at the code, it has not changed.

No error displays in Visual Studio Code.
